packages/commonwealth/server/CommonwealthConsumer/messageProcessors/snapshotConsumer.ts

```diff
@@ -56,25 +89,20 @@ export async function processSnapshotMessage(
     space,
   });
 
-  if (eventType === 'proposal/deleted') {
-    this.log.info(`Proposal deleted, deleting record`);
-    await proposal.destroy();
-
-    StatsDController.get().increment('cw.deleted_snapshot_proposal_record', 1, {
-      event: eventType,
-      space,
-    });
-  }
-
   const associatedCommunities =
     await this.models.CommunitySnapshotSpaces.findAll({
-      where: { snapshot_space_id: space },
+      where: { snapshot_space_id: proposal?.space },
     });
 
   this.log.info(
     `Found ${associatedCommunities.length} associated communities for snapshot space ${space} `
   );
 
+  // Delete the proposal
+  if (eventType === 'proposal/deleted') {
+    await proposal?.destroy();
```

Comment:
I feel like we shouldn't necessarily delete our record of it? Like what if users have already connected the proposal to multiple threads? We probably should store that it _has been deleted_, but not actually delete it from our record.

Reply:
@jnaviask @rvmelkonian @alexyoung23j

Reply:
That works.

Reply:
Which is actually better, since snapshot.org actually deletes it fully lol you can't even fetch the data about it after its been deleted